src/AppInstallerCommonCore/Fonts.cpp

```diff
@@ -542,10 +524,10 @@ namespace AppInstaller::Fonts
             throw std::logic_error("Only WinGet format of font install is supported.");
         }
 
-        if (!context.PackageFiles.has_value() || context.PackageFiles.value().size() == 0)
+        if (context.PackageFiles.size() == 0)
         {
             result.HResult = winrt::hresult(APPINSTALLER_CLI_ERROR_FONT_FILE_NOT_FOUND);
-            AICLI_LOG(Core, Error, << L"Font package has no files: " << context.PackageId.value().c_str() << " - " << result.HResult);
+            AICLI_LOG(Core, Error, << L"Font package has no files: " << context.PackageId.c_str() << " - " << result.HResult);
```

Comment:
There is a helpful stream manipulator that sets things up to output an HRESULT in hex (`AppInstaller::Logging::SetHRFormat`).

Reply:
Ty, updated to use this!